bin/clone_test.sh

```diff
@@ -10,7 +10,7 @@ thisdir=`dirname $realpath`
 . $thisdir/aomp_common_vars
 # --- end standard header ----
 
-EPSDB_LIST=${EPSDB_LIST:-"openmpapps sollve_vv Nekbone goulash BabelStream OvO"}
+EPSDB_LIST=${EPSDB_LIST:-"openmpapps sollve_vv Nekbone goulash BabelStream fortran.babelstream OvO"}
```

Comment:
Unfortunately adding this will not clone what you want. I added support in https://github.com/ROCm-Developer-Tools/aomp/pull/520.